src/components/Atoms/EventsButton/EventsButton.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
```

Comment:
Maybe this is more like a molecule now that it's wrapping Tooltip and CustomButton.

Reply:
Agreed, this is still difficult to guess correctly for me at the moment.